src/panels/lovelace/editor/view-editor/hui-edit-view.ts

```diff
@@ -87,6 +87,16 @@ export class HuiEditView extends LitElement {
     return this.shadowRoot!.querySelector("ha-paper-dialog")!;
   }
 
+  private get _viewConfigTitle(): string {
+    if (!this._config || !this._config.title || this._config.title === "") {
```

Comment:
If there is no config we might want to change the header to `Add new view`? (and an empty string is falsy in javascript so `this._config.title === ""` is not needed.